src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigratePackageDependenciesAndToolsRule.cs

```diff
@@ -203,6 +206,21 @@ private void CleanExistingPackageReferences(ProjectRootElement outputMSBuildProj
                     itemGroup,
                     mergeExisting: true);
             }
+
+            if (project.GetProjectType() == ProjectType.Library && !hasNetStandardReference)
+            {
+                // int is meaningless since the template argument is not being used
+                AddItemTransform<int> transform = new AddItemTransform<int>(
```

Comment:
Can you just use a PackageDependencyInfoTransform instead of creating your own? You just create a PackageInfo (which has three properties, out of which you need to populate two) and call PackageDependencyInfoTransform.

Much cleaner code than this.